src/ConversationRater.res

```diff
@@ -22,18 +22,18 @@ let make = (~conversation, ~onRating) => {
   <div className="space-x-1">
     <RateButton
       icon=#"icon-thumbs-up-alt"
-      onClick={onRating(conversation, Green)}
+      onClick={evt => onRating(conversation, Green, evt)}
```

Comment:
ReScript 11 will now consider every function to be an "uncurried" function by default. Previously, this `onClick` handler did a partial application, which means `onRating(conversation, Green)` would return a new function `(evt) => unit` with `converation` and `Green` being partially applied.

There's a way to define a partially applied function to get back the original behavior, but generally speaking it's not worth the cleverness of auto-curried functions, so let's keep being explicit like in JavaScript here.